resources/js/Shared/Widgets/RemoteWorkList.vue

```diff
@@ -71,7 +71,7 @@
             <template #head>
               <HomeCityIcon
                 class="flex justify-center"
-                size="48"
+                :size="48"
               />
```

Comment:
Why not inline?
```html
<HomeCityIcon class="flex justify-center" :size="48"/>
```

Or even:
```html

```html
<HomeCityIcon class="flex justify-center" :size="48"></HomeCityIcon>
```
```

Reply:
We enforce limits to the one number of attributes/properties per line. That's the `vue/max-attributes-per-line` rule.

Reply:
I don't like it, but okay.